app/actions/remote/channel.ts

```diff
@@ -249,13 +249,14 @@ export async function createChannel(serverUrl: string, displayName: string, purp
     }
 }
 
-export async function patchChannel(serverUrl: string, channelPatch: Partial<Channel> & {id: string}) {
+export async function patchChannel(serverUrl: string, channelId: string, channelPatch: ChannelPatch) {
```

Comment:
Why the additional channelId arg and not the id prop of the object as it was?

Reply:
3/5 for better type clarity and for allowing type declartion (vs forced assertion) when creating the patch.

The standard pattern here seemed to be `serverUrl: string, channelId: string`... as the first two args, and `ChannelPatch` doesn't contain an ID on the server, so this avoids sending extra/unused properties.